moon/includes/modules/cfclogger.moon

```diff
@@ -1,3 +1,5 @@
+AddCSLuaFile!
```

Comment:
should AddCSLuaFile be in an autorun  file instead of here?
if an addon never requires logger serverside then will this never be called?

on cfc this wont ever be a problem but if someone else was using cfclogger for a clientside only addon it might

Reply:
Yeah you're probably right